Error building openssl

## Description
Tried doing `crew upgrade` after a while and I'm now seeing an error building openssl; something about not being able to find libc.

If you executed a specific command:
```shell
crew upgrade
```

Output:
```shell
/usr/local/bin/ld: libcrypto.a(dso_dlfcn.o): in function `dlfcn_globallookup':
dso_dlfcn.c:(.text+0x11): warning: Using 'dlopen' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/local/bin/ld: cannot find /usr/local/lib64/libc.so.6
/usr/local/bin/ld: cannot find /usr/local/lib64/ld-linux-x86-64.so.2
collect2: error: ld returned 1 exit status
make[4]: *** [Makefile.shared:169: link_a.gnu] Error 1
make[4]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2p.tar.gz.dir/openssl-OpenSSL_1_0_2p'
make[3]: *** [Makefile:357: do_linux-shared] Error 2
make[3]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2p.tar.gz.dir/openssl-OpenSSL_1_0_2p'
make[2]: *** [Makefile:310: libcrypto.so.1.0.0] Error 2
make[2]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2p.tar.gz.dir/openssl-OpenSSL_1_0_2p'
make[1]: *** [Makefile:109: shared] Error 2
make[1]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2p.tar.gz.dir/openssl-OpenSSL_1_0_2p/crypto'
make: *** [Makefile:287: build_crypto] Error 1
openssl failed to install: `make V=0 -j4` exited with 2
```

## Versions and system information
chromebrew: `crew --version`
```
chronos@localhost ~/julia $ crew --version
1.1.2
```

ChromeOS: `cat /etc/lsb-release`
```
chronos@localhost ~/julia $ cat /etc/lsb-release
CHROMEOS_ARC_ANDROID_SDK_VERSION=28
CHROMEOS_ARC_VERSION=5151026
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_RELEASE_BOARD=eve-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=0
CHROMEOS_RELEASE_BUILDER_PATH=eve-release/R72-11307.0.0
CHROMEOS_RELEASE_BUILD_NUMBER=11307
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=72
CHROMEOS_RELEASE_DESCRIPTION=11307.0.0 (Official Build) dev-channel eve 
CHROMEOS_RELEASE_KEYSET=mp
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=dev-channel
CHROMEOS_RELEASE_VERSION=11307.0.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=11307.0.0
```
Model: Pixelbook

Architecture: `uname -m`
```
chronos@localhost ~/julia $ uname -m
x86_64
```

--- a/packages/bz2.rb
+++ b/packages/bz2.rb
@@ -7,6 +7,19 @@ class Bz2 < Package
   source_url 'https://fossies.org/linux/misc/bzip2-1.0.6.tar.xz'
   source_sha256 '4bbea71ae30a0e5a8ddcee8da750bc978a479ba11e04498d082fa65c2f8c1ad5'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/bz2-1.0.6-3-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/bz2-1.0.6-3-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/bz2-1.0.6-3-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/bz2-1.0.6-3-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'abfa979b02b0f587c65b7bebb1bc97ba9eda57aec1cb5f3657b6d4f92b90ee65',
+     armv7l: 'abfa979b02b0f587c65b7bebb1bc97ba9eda57aec1cb5f3657b6d4f92b90ee65',
+       i686: '426ec6778bb285ac377f43581a06391ca5a23b533dcfc02cc5c3e03d4239af9f',
+     x86_64: '3ec7418e132350c4989ff2a6856b320d68fbec02913043a6c19f2c025732217f',
+  })
+
   def self.build
     system "make -f Makefile-libbz2_so"
   end

--- a/packages/gdbm.rb
+++ b/packages/gdbm.rb
@@ -8,8 +8,16 @@ class Gdbm < Package
   source_sha256 '86e613527e5dba544e73208f42b78b7c022d4fa5a6d5498bf18c8d6f745b91dc'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gdbm-1.18.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gdbm-1.18.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gdbm-1.18.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gdbm-1.18.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'b7d1eefc792b5250b4a1e1c6cf7845cb965944c5f2f4468b63201e6dc7374d66',
+     armv7l: 'b7d1eefc792b5250b4a1e1c6cf7845cb965944c5f2f4468b63201e6dc7374d66',
+       i686: '60deef61ee53f9aab9a94767a8902ee29920302e113ca115567f3b2d428ae582',
+     x86_64: 'b52cde5502c9bf35fe485776d976d1a44a872e2ffee8964297ac2fe3ef0d064e',
   })
 
   depends_on 'readline7'

--- a/packages/mandb.rb
+++ b/packages/mandb.rb
@@ -55,10 +55,6 @@ class Mandb < Package
 
   def self.postinstall
     puts
-    puts "To finish the installation, set the default MANPATH environment variable:".lightblue
-    puts "echo \"export MANPATH=#{CREW_PREFIX}/man:$MANPATH\" >> ~/.bashrc".lightblue
-    puts "source ~/.bashrc".lightblue
-    puts
     puts "To create the man databases and get apropos working, type 'mandb -c'.".lightblue
     puts
   end

--- a/packages/openssl.rb
+++ b/packages/openssl.rb
@@ -7,6 +7,19 @@ class Openssl < Package
   source_url 'https://github.com/openssl/openssl/archive/OpenSSL_1_0_2p.tar.gz'
   source_sha256 '95ca65a25bdd41e127e5f4054539e8532a46be602b43b44af7c7100172e7cd50'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/openssl-1.0.2p-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/openssl-1.0.2p-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/openssl-1.0.2p-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/openssl-1.0.2p-1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '9d2a1bdeac0d26ec7a22a91e017064edf1799f1783ebb7bce89e92e95b68f043',
+     armv7l: '9d2a1bdeac0d26ec7a22a91e017064edf1799f1783ebb7bce89e92e95b68f043',
+       i686: '0329163f9f9c78433d8b3d22d5a94eccc067ff557810c0fc85d4a8090cacb494',
+     x86_64: 'b61308c6d0cddb3a6e0713e9d47b95382b1c5f0f9495766cec1afa6a07ccf56c',
+  })
+
   depends_on 'bc' => :build             # required for `make test`
 
   def self.build
@@ -47,10 +60,10 @@ class Openssl < Package
     system "rm", "-rf", "#{CREW_DEST_DIR}/etc"
 
     # compress man pages
-    system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/man/man1"
-    system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/man/man3"
-    system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/man/man5"
-    system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/man/man7"
+    system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/share/man/man1"
+    system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/share/man/man3"
+    system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/share/man/man5"
+    system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/share/man/man7"
   end
 
   def self.check

